src/nft/components/bag/BagFooter.tsx

```diff
@@ -454,30 +454,35 @@ export const BagFooter = ({ setModalIsOpen, eventProperties }: BagFooterProps) =
     using_erc20: !!inputCurrency,
     ...eventProperties,
   }
+  const chainAllowed = isSupportedChain(chainId)
```

Comment:
this variable is used once, you could just use `isSupportedChain(chainId)` on L465 instead